src/lib/tames.ts

```diff
@@ -1,25 +1,182 @@
-import { KlasaMessage, KlasaUser } from 'klasa';
+import { userMention } from '@discordjs/builders';
+import { GuildMember, MessageActionRow, MessageButton } from 'discord.js';
+import { increaseNumByPercent, objectEntries, round, Time } from 'e';
+import { KlasaUser } from 'klasa';
 import { APIInteractionGuildMember } from 'mahoji';
-import { Bank, Items, Monsters } from 'oldschooljs';
+import { Bank, Items } from 'oldschooljs';
 import { Item, ItemBank } from 'oldschooljs/dist/meta/types';
+import { ChambersOfXeric } from 'oldschooljs/dist/simulation/misc';
 
 import { collectables } from '../mahoji/lib/abstracted_commands/collectCommand';
 import { mahojiUsersSettingsFetch } from '../mahoji/mahojiSettings';
 import { getSimilarItems } from './data/similarItems';
-import killableMonsters, { effectiveMonsters } from './minions/data/killableMonsters';
+import killableMonsters from './minions/data/killableMonsters';
+import { customDemiBosses } from './minions/data/killableMonsters/custom/demiBosses';
+import { KillableMonster } from './minions/types';
 import { prisma, trackLoot } from './settings/prisma';
 import { runCommand } from './settings/settings';
-import { itemNameFromID, roll } from './util';
-import { createCollector } from './util/createCollector';
+import { assert, channelIsSendable, itemNameFromID, roll } from './util';
 import getOSItem from './util/getOSItem';
-import { collectors } from './util/handleTripFinish';
-import { makeBankImage } from './util/makeBankImage';
-import { sendToChannelID } from './util/webhook';
+import { makeBankImageKlasa } from './util/makeBankImage';
+import resolveItems from './util/resolveItems';
 import { Tame, tame_growth, TameActivity, User } from '.prisma/client';
 
+export enum TameSpeciesID {
+	Igne = 1,
+	Monkey = 2
+}
+
+export const igneArmors = [
+	{
+		item: getOSItem('Dragon igne armor'),
+		coxDeathChance: 85,
+		foodReduction: 5
+	},
+	{
+		item: getOSItem('Barrows igne armor'),
+		coxDeathChance: 75,
+		foodReduction: 8
+	},
+	{
+		item: getOSItem('Volcanic igne armor'),
+		coxDeathChance: 65,
+		foodReduction: 10
+	},
+	{
+		item: getOSItem('Justiciar igne armor'),
+		coxDeathChance: 50,
+		foodReduction: 15
+	},
+	{
+		item: getOSItem('Drygore igne armor'),
+		coxDeathChance: 30,
+		foodReduction: 22
+	},
+	{
+		item: getOSItem('Dwarven igne armor'),
+		coxDeathChance: 10,
+		foodReduction: 27
+	},
+	{
+		item: getOSItem('Gorajan igne armor'),
+		coxDeathChance: 5,
+		foodReduction: 33
+	}
+].map(i => ({ ...i, tameSpecies: [TameSpeciesID.Igne], slot: 'equipped_armor' as const }));
+
+export type TameKillableMonster = {
+	loot: (opts: { quantity: number; tame: Tame }) => Bank;
+	deathChance?: (opts: { tame: Tame }) => number;
+	oriWorks?: false;
+	mustBeAdult?: true;
+	minArmorTier?: Item;
+} & Omit<KillableMonster, 'table'>;
+
+function calcPointsForTame(tame: Tame) {
+	const lvl = tame.max_combat_level;
+	if (lvl < 75) return 10_000;
```

Comment:
**This is just a suggestion, not an issue.**

While i'm not saying cater to ironmen, I do think they might be getting overly hit with this, as you can go 3, 4 eggs without getting 80+ if ur just a little unlucky.

I might scale it 15k => 24k, this is just a suggestion